code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -490,6 +490,15 @@
 	toxpwr = 0.5
 	taste_description = "bad cooking"
 
+//MonkeStation Edit: Bad Food can cause disease
+/datum/reagent/toxin/bad_food/reaction_mob(mob/living/M, method = INGEST, reac_volume, show_message, touch_protection)
+	. = ..()
+	if((ishuman(M) || ismonkey(M)) && prob(20))
+		var/datum/disease/advance/plague = new /datum/disease/advance/random(rand(2,3), 9, 0, /datum/symptom/vomit)
```

Comment:
just checking, what do the new args mean? are they levels for different symptoms?

Reply:
Number of symptoms, Max level, Minimum level, Required symptom included.

Thus, you get between 2 and 3 symptoms, a maximum symptom level of 9 (highest existing), a minimum symptom level of 0 (lowest existing) and every disease generated must have the Vomiting symptom added to it.
Vomiting because food poisoning is the theme, but the rest of the disease is going to be the wacky effects of the chef being a terrible cook. 
"WHY AM I PUKING UP BEES, CHEF?"